### DISCUSSION TITLE
What are the main benefits of React Native for Web?

### DISCUSSION THREAD

**Discussion by vongohren** [necolas/react-native-web#2033] [Q&A]
Hi! We have an internal discussion on the main benefits of RNW. Does it make sense to use RNW only when you have a React Native app? Or does it exists any other reasons?

We use mobile.twitter.com as an example for discsussion and wonder what the benefit of this is, as it seems the Twitter app is native? How do one gain speed and benefits in a situation like this?

**Comment by intergalacticspacehighway** [comment]
I found these useful
- Built in Gesture handler (PanResponder)
- Imperative Animated API
- Better RTL APIs
- Built in CSS in JS
- Makes it easy to achieve functionality like pull to refresh via ScrollView.
- Built in useful APIs (FlatList, SectionList, Modal, color mode, clipboard)

and the obvious one is code sharing if you're already using React Native.
